packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -78,39 +79,73 @@ export const styles = theme => {
         top: 'initial',
         bottom: 0,
         width: 2,
+        height: '100%',
       },
     },
     /* Styles applied to the track element before the thumb. */
     trackBefore: {
       zIndex: 1,
       left: 0,
-      transition: trackTransitions,
+      transformOrigin: 'left bottom',
+      '&$rtl': {
+        // left gets rewritten to right which is actually what we want
+        transformOrigin: 'left bottom',
```

Comment:
As you already wrote in the comments `jss-rtl` already takes care of this. The `&$rtl` rule is redundant.
![slider-rtl-redundant](https://user-images.githubusercontent.com/12292047/47291484-1bd5a680-d604-11e8-93dc-8c212b80b0df.png)